lib/middleware/tagged_logging.rb

```diff
@@ -20,5 +22,14 @@ def call(env)
       ::Logging.mdc.delete('request')
       ::Logging.mdc.delete('session')
     end
```

Comment:
One more idea - can you send the full UUID to the log at the moment the session is opened (user sign in) at the INFO level? If we get the prefix conflict, it is possible to identify who was who. User id could be logged too with this special entry.